tools/code_format/config.yaml

```diff
@@ -295,3 +295,5 @@ visibility_excludes:
 - source/extensions/transport_sockets/common/BUILD
 - source/extensions/udp_packet_writer/default/BUILD
 - source/extensions/udp_packet_writer/gso/BUILD
+- source/extensions/path/match/pattern_template/BUILD
```

Comment:
Note: Senior Maintainer needs to approve this section explicitly

Reply:
Not sure why presubmit is still tagging visibility changes... @zuercher Any ideas?